### PR TITLE
#166495422 Create a home page

### PR DESCRIPTION
**Pull request by niyobobo**
### What does this PR do?
Setting up a sample home page that has `navbar`, `navigation` and `footer` components
### Description of Task to be completed
- Removing the existing dummy components template.
- Creating a new welcome page based on the mockup we have.
- Creating other reusable components like `footer` and `navigation header`.
- Writing unit tests of those components.
### How should this be manually tested?
- Clone [repo](https://github.com/andela/ah-kgl-avengers-frontend/) and checkout to branch `ft-home-page-166495422`.
- Install dependencies by running `npm install` script.
- Run `npm start` script to run the application in the browser.
- Run `npm test` to check all test.
### Any background context you want to provide?
- The application runs on port `8000` make sure no other process runs on the same port.
- Home page body has no content. Navigation and Footer, one is underneath another but when articles loaded in Home component the footer will span down. 
### What are the relevant pivotal tracker stories?
[#166495422](https://www.pivotaltracker.com/n/projects/2321840/stories/166495422)
### Screenshots (if appropriate)
**Full screen**
image
**Mobile version**

### Questions:
N/A

### CODE REVIEW

**Comment by avpaul** [comment]
@niyobobo  In the PR description, there is a typo:
`Install dependencies by run npm install script.` should be `Install dependencies by running npm install script.`

**Review by avpaul — CHANGES_REQUESTED** [review]
Great work @niyobobo 👏 👏  But the design breaks on smaller devices, which I think is because of the footer. Plus the second navigation, the one with categories should be made responsive(consider making it a dropdown).

**Comment by niyobobo** [comment]
> @niyobobo In the PR description, there is a typo:
> `Install dependencies by run npm install script.` should be `Install dependencies by running npm install script.`

Thank you @avpaul for the feedback. I have fixed it.

File: src/components/functional/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  Container, Nav, NavItem, NavLink,
+} from 'reactstrap';
+
+const Footer = () => (
```
Comment (ezkemboi) [inline src/components/functional/footer.jsx:6]: The footer is still not responsive. 
Screen Shot 2019-06-12 at 09 36 58

File: src/components/functional/navBar.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    const { isToggled } = this.state;
+    this.setState({ isToggled: !isToggled });
+  }
+
+  render() {
+    const { isToggled } = this.state;
+    return (
+      <Navbar dark className="nav-top shadow-sm p-3" expand="sm">
+        <Container>
+          <NavbarBrand className="nav-logo">
```
Comment (ezkemboi) [inline src/components/functional/navBar.jsx]: I would love this navbar logo to be clickable. When I click navbar, it takes me to `/` page, no matter the page I am. 
Make the whole logo image and text clickable and redirect always to homepage.

**Review by ezkemboi — CHANGES_REQUESTED** [review]
I have left some review on the PR

**Comment by ezkemboi** [comment]
Also, please update screenshots section of the PR @niyobobo. Thank you.

**Review by ezkemboi — APPROVED** [review]
Changes made are satisfactory on my side.

**Review by avpaul — APPROVED** [review]

**Review by fridolinho — APPROVED** [review]

**Pull request merged**